docker/1.6.0/py3/Dockerfile.cpu

```diff
@@ -77,6 +77,7 @@ RUN ${PIP} install --no-cache --upgrade \
     requests==2.22.0 \
     scikit-learn==0.20.4 \
     dgl==0.4.1 \
+    git+git://github.com/dmlc/gluon-nlp.git@7f52dc05367b4640833abf2d767c98e4719395af \
```

Comment:
is there a reason this is pinned to a specific commit? are we not planning on using a pypi released version of gluon nlp?

Reply:
Yes.  As DLC relies on MXNet pre-release version, we need to use GluonNLP pre-release version for compatibility.

Reply:
Instead of pinning a specific commit here, can you add a tag or something corresponding to the DLC release? This will help give us more context on what's installed and provide users more information on what is being installed/why in regards to gluon nlp when they navigate to the tag of the gluon nlp repo